cluster/kubectl.sh

```diff
@@ -13,11 +13,11 @@ fi
 if [ "x$1" == "xspice" ]; then
         viewer=${SPICE_VIEWER:-remote\-viewer}
         if [ "x$3" == "x--details" ]; then
-            curl http://${master_ip}:8184/apis/kubevirt.io/v1alpha1/namespaces/default/vms/$2/spice  -H"Accept:text/plain"
+            curl -s http://${master_ip}:8184/apis/kubevirt.io/v1alpha1/namespaces/default/vms/$2/spice  -H"Accept:text/plain"
```

Comment:
Was that there before already? Why not showing the errors on stdout? They are valid imho.